File: joao pedro/Problema0394/Problema0394.java
import java.util.Scanner;

public class Problema0394 {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        double [][] gasto=new double[24][2];
        double maiorGasto=0;
        int ano=0,mes=0;
        double consumo1=0;
        double consumo2=0;
        double medio1=0;
        double medio2=0;
        for(int i=0;i<2;i++){
            for(int j=0;j<12;j++){
                System.out.println("Digite o gasto do mes "+j+" no ano "+i+": ");
                gasto[j][i]=Double.valueOf(input.nextLine());
                if(gasto[j][i]>maiorGasto){
                    maiorGasto=gasto[j][i];
                    ano=i;
                    mes=j;
                }
                if(i==0){
                    consumo1=gasto[j][i]+consumo1;
                    
                } else{
                    consumo2=gasto[j][i]+consumo2;
                }
            }
        }
       System.out.println("Consumo medio em 2020 (ano 0): "+consumo1/12);
       System.out.println("Consumo medio em 2022 (ano 1): "+consumo2/12);
        System.out.println("Maior gasto de energia (Mes/Ano): "+mes+"/"+ano);

        System.out.println("Consumo em 2020 (ano 0): "+consumo1);
        System.out.println("Consumo em 2022 (ano 1): "+consumo2);
    }
}

File: joao pedro/Problema0394/Problema0394.py
gasto = [[0.0] * (2) for _ in range(24)]
maiorGasto = 0
ano = 0
mes = 0
consumo1 = 0
consumo2 = 0
medio1 = 0
medio2 = 0
x=0
i = 0
while (i < 2) :
    j = 0
    while (j < 12) :
        print("Digite o gasto do mes " ,j, " no ano " ,i, ": ")
        gasto[j][i]=float(input(""))
        
        if (gasto[j][i] > maiorGasto) :
            maiorGasto = gasto[j][i]
            ano = i
            mes = j
        if (i == 0) :
            consumo1 = gasto[j][i] + consumo1
        else :
            consumo2 = gasto[j][i] + consumo2
        j+=1
    i+=1
    

        
print("Consumo medio em 2020 (ano 0): " ,consumo1 / 12)
print("Consumo medio em 2022 (ano 1): " ,consumo2 / 12)
print("Maior gasto de energia (Mes/Ano): " ,mes, "/" ,ano)
print("Consumo em 2020 (ano 0): " ,consumo1)
print("Consumo em 2022 (ano 1): " ,consumo2)